db/migrations/2023080311350006-add-comments-github-to-comments-table.js

```diff
@@ -0,0 +1,165 @@
+const { Op } = require("sequelize");
+const Octokit = require("octokit").Octokit;
+const ChainModel = require("../models/chain.model");
+const NetworkModel = require("../models/network.model");
+const IssueModel = require("../models/issue.model");
+const TokenModel = require("../models/tokens.model");
+const RepositoryModel = require("../models/repositories.model");
+const PullRequestModel = require("../models/pullRequest.model");
+const MergeProposalModel = require("../models/mergeproposal");
+const CuratorsModel = require("../models/curator-model");
+const BenefactorModel = require("../models/benefactor.model");
+const DisputeModel = require("../models/dispute-model");
+const UserPaymentsModel = require("../models/user-payments");
+const DeveloperModel = require("../models/developer.model");
+const CommentsModel = require("../models/comments.model");
+const UserModel = require("../models/user");
+
+const { SKIP_MIGRATION_SEED_COMMENTS_DATE_GITHUB, NEXT_GH_TOKEN } = process.env;
+const BOT_NAME = 'bepro-bot'
+
+async function handleAddComments(comment, id, type, prId){
+  const regexGithubLogin = /@(\w+)/;
+  const bodyGithubLogin = regexGithubLogin.exec(comment.body);
+
+  const getCommentCreateData = (userId, userAddress, body) => ({
+    userId,
+    userAddress,
+    comment: body,
+    issueId: id,
+    hidden: false,
+    type,
+    created_at: comment.created_at,
+    updated_at: comment.updated_at,
+    ... prId ? { deliverableId: prId } : null
+  }) 
+
+  const getUser = (name) => UserModel.findOne({
+    where: {
+      githubLogin: name
+    }
+  })
+
+  if(bodyGithubLogin && comment.user.login === BOT_NAME){
```

Comment:
we don't need the `bodyGithubLogin` outside of this scope, so we can simply do `if (comment.body.startsWith('@') && coment.user.login === BOT_NAME)` and then, inside, you grab the value to a variable:

```js
const user = await getUser(comment.body.split(' ')[0].replace('@',''))
```